resource/ui/ts/models/timeseries.ts

```diff
@@ -43,68 +44,183 @@ module Models {
         /**
          * QueryResultSet matches the successful output of the /ts/query
          * endpoint. This needs to be kept in sync with the
-         * TimeSeriesQueryResult protobuffer message on the server.
+         * TimeSeriesQueryResponse protobuffer message on the server.
          * (/protos/timeseries.proto).
          */
         export interface QueryResultSet {
```

Comment:
Added a TODO to move all protobuffer-based interfaces into a common, separate module.